src/main/java/com/cascadebot/cascadebot/messaging/MessageType.java

```diff
@@ -7,15 +7,15 @@
 
 import com.cascadebot.shared.SharedConstants;
 
-import java.awt.*;
+import java.awt.Color;
 
 public enum MessageType {
 
-    INFO(Color.CYAN, "\u2139"), // ℹ
-    SUCCESS(Color.GREEN, "\u2705"), // ✅
-    WARNING(Color.YELLOW, "\u26A0"), // ⚠
-    MODERATION(Color.WHITE, "\uD83D\uDC6E"), // 👮
-    DANGER(Color.RED, "\u274C"), // ❌
+    INFO(Color.CYAN, "\u2139"), // Information icon
```

Comment:
I feel like we should also include the unicode character

Reply:
Okay